### ISSUE TITLE
"Modules" in frontend view

### ISSUE THREAD

**Issue by javierglez-colillas** [weDevsOfficial/wp-project-manager#432]
Hi Dev Team, 

In the frontend, it is strange that if the user does not have the admin role he can activate/deactivate modules that affect ALL projects.

This button will be hidden or only accessible from "Settings"

**Comment by usman244718** [comment]
Yes, I am available. Please share some details about the task. I am eager to collaborate with you. with 6+ years experience, please dm me if you want my portfolio and resume.
[#fullstack](https://twitter.com/hashtag/fullstack?src=hashtag_click) development, [#graphic](https://twitter.com/hashtag/graphic?src=hashtag_click) designing

**Comment by usman244718** [comment]
share me your budget I will do it for you

**Comment by arifulhoque7** [comment]
Addressed in the new UI revamp — module activation/deactivation is no longer exposed to non-admin users in the frontend. Closing as resolved by the UI revamp.
